api/v2/checluster_types.go

```diff
@@ -873,3 +873,9 @@ func (c *CheCluster) IsEmbeddedOpenVSXRegistryConfigured() bool {
 	}
 	return openVSXURL == ""
 }
+
+// IsCheBeingInstalled returns true if the Che version is not set in the status.
+// Basically it means that the Che is being installed since the Che version is set only after the installation.
+func (c *CheCluster) IsCheBeingInstalled() bool {
+	return c.Status.CheVersion == ""
+}
```

Comment:
is it the most reliable way for checking if the project / product is installed on a cluster?

Reply:
If installation completed, then version is set.
So, I would say that I don't know the better way to detect it.